apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -215,6 +206,45 @@ const TooltipContent = ({
   );
 };
 
+export const PropertyTooltip = ({
+  openWithClick = false,
+  title,
+  description,
+  properties,
+  style,
+  onReset,
+  children,
+}: {
+  openWithClick?: boolean;
+  title?: string;
+  description?: React.ReactNode;
+  properties: StyleProperty[];
+  style: StyleInfo;
+  onReset?: undefined | (() => void);
+  children: ReactElement;
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <Tooltip
+      open={isOpen}
+      onOpenChange={setIsOpen}
+      onClick={() => setIsOpen(false)}
+      triggerProps={openWithClick ? { onClick: () => setIsOpen(true) } : {}}
```

Comment:
- not sure why there is both onClick and triggerProps.onClick
- is there a particular reason to not pass onClick with triggerProps always and then do nothing is openWithClick is false? it would read more straight forward

Reply:
I wanted to isolated open state in tooltip to not add in every `<PropertyTooltip>` usage.